### DISCUSSION TITLE
New piano song suggestion! :D - I NEED U by BTS

### DISCUSSION THREAD

**Discussion by xXMidnight** [Monika-After-Story/MonikaModDev#7810] [general - piano notes]
### Verse 1:

Fall
[t u]

Everything
y u o u

Fall
[r y]

Everything
u y t e

Fall
[q e]

Everything
t y o u

흩어지네 (heut-eojine)
u p o u

Fall
[t u]

Everything
y u o u

Fall
[r y]

Everything
u y t e

Fall
[q e]

Everything
t y o u

떨어지네 (tteol-eojine)
u p o u

### Verse 2:

I need you girl
] [ p p

왜 (wae)
p

혼자 사랑하고 (honja salanghago)
o p p o p u

혼자서만 이별해 (honjaseoman ibyeolhae)
o p p o p p ]

I need you girl
] [ p p

왜 (wae)
p

다칠 걸 알면서 (dachil geol almyeonseo)
o p p o p u

자꾸 네가 필요해 (jakku nega pil-yohae)
o p p o p p ]

I need you girl
] [ p p

넌 아름다워 (neon aleumdawo)
e t o o u

I need you girl
] [ p p

너무 차가워 (neomu chagawo)
e t o o u

I need you girl
] [ p p

I need you girl
] [ p p

I need you girl
] [ p p

e p o u

### Verse 3:

Fall
[t u]

Everything
y u o u

Fall
[r y]

Everything
u y t e

Fall
[q e]

Everything
t y o u

흩어지네 (heut-eojine)
u p o u

Fall
[t u]

Everything
y u o u

Fall
[r y]

Everything
u y t e

Fall
[q e]

Everything
t y o u

떨어지네 (tteol-eojine)
u p o u

**Comment by xXMidnight** [comment]
I guess you can dissect it yourself if you've heard it before. I think I'll dissect it in the morning, since it's currently 9:30pm for me and I need to go sleep. Please do this song! :D

also if you haven't noticed there's a [t u] there and it's because you have to play both notes at the same time

**Comment by 20RaVen23** [comment]
Don't mind me, just tagging the main issue for submod/piano notes and etc.
#3765

**Comment by xXMidnight** [comment]
> Don't mind me, just tagging the main issue for submod/piano notes and etc.
> #3765

too late
